### PR TITLE
Pos transaction system using arduino

### PR DESCRIPTION
**Pull request by lalith1403**
I have included the following images: 

1. wireframe_1.jpg
2.wireframe_2.jpg
3.wireframe_3.jpg
4.arduino.jpg

The code has been checked for and detailed instructions are given. Kindly review the details and let me know.

### CODE REVIEW

**Comment by hectorkambow** [comment]
Good afternoon 👋 @glennslaven 🚀 - wanted to ask if you could give this article a tech review before we publish - whenever you have a moment 
Thank you

**Comment by hectorkambow** [comment]
@ninjaginja  - similar merge issue 🤔

File: articles/pos-system-using-arduino-and-python/index.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+
+### Hardware
+#### RFID Tag and Scanner
+”RFID” is an abbreviation for Radio Frequency Identification. RFID uses electromagnetic fields to automatically identify and track tags attached to objects. The tags contain electronically stored information. Passive tags collect energy from a nearby RFID reader’s interrogating radio waves.
+
+The RFID module used is MFRC522. The RFID module is connected to
+Arduino. Here we are reading the UID number of a person’s ID card using
+RFID scanner. We load the UID value onto the RFID tag.
+
+![alt text](/engineering-education/pos-system-using-arduino-and-python/arduino.jpg "Title")
```
Comment (glennslaven) [inline articles/pos-system-using-arduino-and-python/index.md:42]: Actual alt text & title?

**Comment by hectorkambow** [comment]
wireframe_1

**Pull request merged**